src/styles/gnb/index.tsx

```diff
@@ -1,6 +1,10 @@
 import styled from "styled-components";
 
-const Wrapper = styled.nav`
+interface GnbStyleProps {
+  $location: string;
+}
+
+const Wrapper = styled.nav<GnbStyleProps>`
```

Comment:
지금과 같은 경우에 하나의 props만을 받기 위해 Props를 선언해서 사용하는데 이렇게 해볼 수 있을 것 같습니다.

```suggestion
const Wrapper = styled.nav<{ $location: string; }>`
```